### DISCUSSION TITLE
Will it support Material Design 3.0?

### DISCUSSION THREAD

**Discussion by CeSun** [AvaloniaCommunity/Material.Avalonia#225] [General]
3.0 looks more concise and elegant, and there is already a supported control library on maui： 3.0 looks more concise and elegant, and there is already a supported control library on maui

**Comment by SKProCH** [comment]
Hello, its been discussed here: https://github.com/AvaloniaCommunity/Material.Avalonia/issues/183

Current status:
> Not planned yet, due to lack of developer time to support what we have. Perhaps in the future, or if someone wants to implement it themselves and send a PR/help in some other way.
